CHANGES.md

```diff
@@ -1,3 +1,7 @@
+# Unreleased
+- [FIX] Enabled `reduce` and other reduce related parameters to be set when using
+  `MultipleRequestBuilder`.
```

Comment:
After reading back over our API and the semver spec knowing that we shipped 2.0 without reduce on the `MultipleRequestBuilder` interface, I think we need to mark it as `IMPROVED` or `NEW` and release it as a minor.  Because it effectively adds new function. I'm not sure how it would apply to someone who is implementing the interface, but I've convinced myself that it isn't an issue, after reading the spec.